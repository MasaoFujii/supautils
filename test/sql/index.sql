select supa.index(array['a','b','c'], 'c');
