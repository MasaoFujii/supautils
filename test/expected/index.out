select supa.index(array['a','b','c'], 'c');
 index 
-------
     3
(1 row)

